pyiron_atomistics/lammps/control.py

```diff
@@ -607,6 +610,14 @@ def calc_md(
                 job_name=job_name
             )
 
+        if msd:
+            self.modify(
+                compute___1="all msd com yes",
```

Comment:
`compute___1` seems like a super dangerous name likely to lead to conflict. Rather `compute___msd`?